src/main/java/blackjack/domain/gameplayer/Cards.java

```diff
@@ -0,0 +1,62 @@
+package blackjack.domain.gameplayer;
+
+import blackjack.domain.card.Card;
+import blackjack.domain.card.CardType;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class Cards {
+
+    private static final int BLACK_JACK = 21;
+
+    private final List<Card> cards;
+
+    public Cards() {
+        this.cards = new ArrayList<>();
+    }
+
+    public void receiveCard(final Card card) {
+        cards.add(card);
+    }
+
+    public List<Card> toList() {
+        return Collections.unmodifiableList(cards);
+    }
+
+    public int calculateCards() {
+        final int sumOfCards = getSumOfCards();
+        final int aceCount = getAceCount();
+
+        if (sumOfCards > BLACK_JACK && aceCount > 0) {
+            return getBestSumWithAce(sumOfCards);
+        }
+        return sumOfCards;
+    }
+
+    private int getAceCount() {
+        return (int) cards.stream()
+            .filter(card -> card.getCardType().isAce())
+            .count();
+    }
+
+    private int getSumOfCards() {
+        return cards.stream()
+            .map(Card::getCardType)
+            .mapToInt(CardType::getPoint)
+            .sum();
+    }
```

Comment:
메서드 호출되는 순서대로 위치시켜주세요! 🙏 

[참고 자료](https://nesoy.github.io/articles/2018-01/CleanCode-Form)

```suggestion
   private int getSumOfCards() {
        return cards.stream()
            .map(Card::getCardType)
            .mapToInt(CardType::getPoint)
            .sum();
    }
    
    private int getAceCount() {
        return (int) cards.stream()
            .filter(card -> card.getCardType().isAce())
            .count();
    }
```

Reply:
헉 메소드 순서까지 신경써주시다니 ㅠㅠ 꼼꼼한 리뷰 감사드립니다!!